lib/launcher.js

```diff
@@ -26,13 +26,36 @@ const baseBrowserDecoratorFactory = function (
   }
 }
 
-function Launcher (server, emitter, injector) {
-  this._browsers = []
-  let lastStartTime
+class Launcher {
+  constructor (server, emitter, injector) {
+    this._server = server
+    this._emitter = emitter
+    this._injector = injector
+    this._browsers = []
+    this._lastStartTime = null
+
+    this.launch.$inject = [
```

Comment:
This bit is real mysterious to most readers. Can we add a comment like:
```
// Attach list of dependency injection parameters to methods.
```

Reply:
Done